storage/range.go

```diff
@@ -1781,6 +1868,14 @@ func loadAppliedIndex(eng engine.Engine, raftID int64) (uint64, error) {
 	return appliedIndex, nil
 }
 
+func loadLeaderLease(eng engine.Engine, raftID int64) (*proto.Lease, error) {
+	lease := &proto.Lease{}
+	if _, err := engine.MVCCGetProto(eng, engine.RaftLeaderLeaseKey(raftID), proto.ZeroTimestamp, true, nil, lease); err != nil {
```

Comment:
this won't error but just return an empty lease if nothing had previously been persisted?